src/models/document/base-document-content.ts

```diff
@@ -822,6 +822,7 @@ export const BaseDocumentContentModel = types
         const documentEnv = getEnv(self)?.documentEnv as IDocumentEnvironment | undefined;
         const appConfig = documentEnv?.appConfig;
 
+        // TODO: title-refactor: remove title from addTileOptions?
         const newContent = contentInfo?.defaultContent({ title, url, appConfig });
```

Comment:
With the way that titles have been stored for a while now, it isn't really possible for the defaultContent to "set" the title. This is because the defaultContent is a "parentless" tile content model, and the title is stored on the parent TileModel. As far as we can tell the title passed here was silently be ignored. However we didn't remove it because we haven't reviewed the Table tile.